ZigbeeNet/CC/CCZnp.cs

```diff
@@ -153,26 +60,24 @@ private bool _resetting
                 _reset = value;
             }
         }
-        private Timer _resetTimeout;
-        private Timer _timeout;
-        private ZpiSREQ _sreqRunning;
 
-        public CCZnp()
-        {
-            _requestQueue = new ConcurrentQueue<ZpiObject>();
-        }
+        public event EventHandler Ready;
+        public event EventHandler Closed;
+        public event EventHandler ResetDone;
 
+        public event EventHandler<ZpiObject> AsyncResponse;
+        
         public void Init(string port, int baudrate = 115200)
         {
             ZpiMeta.Init();
             ZdoMeta.Init();
 
-            Unpi = new Unpi(port, baudrate, 1);
-            //Unpi.DataReceived += Unpi_DataReceived;
-            //Unpi.Opened += Unpi_Opened;
-            //Unpi.Closed += Unpi_Closed;
+            unpi = new UnifiedNetworkProcessorInterface(port, baudrate, 1);
+            //unpi.DataReceived += Unpi_DataReceived;
```

Comment:
The Event will not longer be assigned, but is used bellow. Is there any other solution planned?

Reply:
Uhm, yeah?

Reply:
Ok, wrong words by me. I had better to say "what will be your solution" :-)